.github/workflows/python-bindings.yml

```diff
@@ -0,0 +1,76 @@
+on:
+  push:
+    tags:
+      - 'accesskit_python-v*'
+
+name: Publish Python bindings
+
+jobs:
+  build-wheels:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: macos-latest
+            python-arch: x64
+            rust-target: x86_64
+          - os: macos-latest
+            python-arch: x64
+            rust-target: universal2-apple-darwin
+          - os: ubuntu-latest
+            python-arch: x64
+            rust-target: x86_64
+          - os: ubuntu-latest
+            python-arch: x64
+            rust-target: i686
+            skip-wheel-installation: true
+          - os: windows-latest
+            python-arch: x64
+            rust-target: x64
+          - os: windows-latest
+            python-arch: x86
+            rust-target: x86
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.7
+          architecture: ${{ matrix.python-arch }}
+      - uses: dtolnay/rust-toolchain@stable
+      - name: Build wheel
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.rust-target }}
+          manylinux: auto
+          args: --release --out dist --sdist
```

Comment:
It's my understanding (I could be wrong) that this will only build the wheel for the python version specified in the `actions/python@v4` action (in this case `3.7`). Getting a good matrix can be rough. In the past, [I've used `pypa/cibuildwheel` as seen here](https://github.com/infinyon/fluvio-client-python/blob/7d10bdb7c55ff720afcbd100050ca5d4d0198ffb/.github/workflows/publish.yml#L9-L71) which definitely has some negatives (the whole build runs in a docker container which can have different libraries than the host).

Reply:
You are right. In fact, here is the link to the download page of a test upload I did: https://test.pypi.org/project/accesskit/#files

Reply:
TIL about test.pypi.org. Cool.

Well, I don't work on the Python client for Fluvio anymore (and it looks like there hasn't been a release in a while) but it's my recollection that `pypa/cibuildwheel` action took a long time (~30 minutes) because each of the python versions was done in serial.

One thing that I also recall being a pretty rough time was if there's OpenSSL 3 in the dependency tree. This uses a different version of perl than OpenSSL 1 and the pypa images didn't have it. I'm currently failing at finding the github issue for this :/

You might be able to get away with just adding a `python-version` list to your matrix and have them run in parallel. It's unclear what kind of target OS/arch/python version support you're looking for. [`pypa/cibuildwheel` has a lot of python versions and targets](https://github.com/pypa/cibuildwheel#what-does-it-do).

Reply:
No, zbus doesn't depend on OpenSSL. On Linux, we don't depend on any C libraries other than the base system libraries (libc, pthread, etc.)

Reply:
I wasn't 100% sure as I am not familiar with D-Bus authentication mechanism, but no OpenSSL involved here thankfully!

Reply:
Coming back to this: we actually only need to build wheels for the oldest Python version we want to support because I'm using the limited ABI3 feature which provide backward compatibility. Since we are not doing much from the Python side, it's best to enable it and only build for, let's say, Pyton 3.9.